src/www/ui/api/Helper/UploadHelper.php

```diff
@@ -821,6 +821,9 @@ public function getUploadLicenseList($uploadId, $agents, $printContainers, $bool
         $responseList[] = $responseRow->getArray();
       }
     }
-    return $responseList;
+    $Offset = $Page * $Limit;
+    $paginatedResponseList = array_slice($responseList, $Offset, $Limit);
+    $paginatedResponseListSize = sizeof($paginatedResponseList);
```

Comment:
`$paginatedResponseListSize` should represent the total size of array, not the size of response. Otherwise client will never know how many pages it can fetch.